### DISCUSSION TITLE
Splitting code into different files

### DISCUSSION THREAD

**Discussion by IlyaDhtvzGhbltn** [cake-build/cake#3704] [Q&A] [answered]
Hi!
I have been using build cake for a long time and my cake file includes more than 1000 lines.
Is there some way to split code into several ***.cake** files ?

**Comment by nils-a** [comment, marked as answer]
Yes, you can have multiple Cake files and then load them all together using the [`#load` preprocessor directive](https://cakebuild.net/docs/writing-builds/preprocessor-directives/load).

There was also a [nice blog post on that topic some time ago](http://www.leerichardson.com/2021/02/he-one-thing-i-wish-id-known-before.html).

**Comment by pascalberger** [comment]
See also the [Reusing Builds](https://cakebuild.net/docs/writing-builds/reusing-builds) documentation topic, which explains how you can split and share code across different builds.
